src/main/java/net/spring/controller/AppController.java

```diff
@@ -0,0 +1,55 @@
+package net.spring.controller;
+
+import net.spring.entity.Student;
+import net.spring.service.StudentService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.ModelAttribute;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.servlet.ModelAndView;
+
+import javax.validation.Valid;
+import java.util.List;
+
+@Controller
+public class AppController {
+    @Autowired
+    private StudentService service;
+
+    @RequestMapping("/")
```

Comment:
Please look at  annotations 
@GetMapping
@PostMapping
@PutMapping
@DeleteMapping
@PatchMapping
and use them
https://www.baeldung.com/spring-requestmapping